charts/rasa-x/templates/rasa-x-deployment.yaml

```
@@ -41,10 +41,10 @@ spec:
           initialDelaySeconds: {{ .Values.rasax.initialProbeDelay }}
           failureThreshold: 10
         env:
-        {{- if .Values.debugMode }}
+        {{ if .Values.debugMode }}
```

let's keep this to avoid unnecessary newlines
```suggestion
        {{- if .Values.debugMode }}
```